app/layout.tsx

```diff
@@ -13,8 +13,8 @@ const geistMono = Geist_Mono({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "PawRide - Uber for Dogs",
```

Comment:
Yo, using a hard-coded title and description? Bruh, make it dynamic with props so it can change with the page content. Just a lil touch to make it more aesthetic 🚀